src/main.rs

```diff
@@ -73,6 +73,11 @@ struct UnzipArgs {
     #[arg(short = 'd', long, value_name = "DIRECTORY")]
     output_directory: Option<PathBuf>,
 
+    /// Password to decrypt encrypted zipfile entries (if any).
+    /// Encryption is assumed to be the ZipCrypto algorithm.
```

Comment:
Are you sure this sentence is correct? It looks to me as though the underlying library supports either ZipCrypto or AES encryption.

I'm wondering about the pros and cons of adding a test for AES encryption too. Pros: thoroughness. Cons: it's really a test of the underlying library, not ripunzip, and it might require enabling additional features in it.

Overall, if you've any doubt about whether AES is supported, please add a test! Otherwise, don't bother - but please clarify this doc comment either way.

Sorry for not spotting this on the first pass.

Reply:
All good, this was a good pickup. Thank you.

I've clarified the comment.

I've opted not to add a test for AES encrypted zipfiles because v0.6.6 of the zip library doesn't seem to have a nice interface to do this. I also didn't want to include a pre-made zip file because it's a bit suspicious finding a random zip file in source code.

I have tested it separately though - i.e. using the binary on a aes encrypted zipfile - yes it does work.